runelite-client/src/main/java/net/runelite/client/plugins/discord/DiscordPlugin.java

```diff
@@ -381,7 +381,14 @@ private void checkForAreaUpdate()
 			return;
 		}
 
-		final DiscordGameEventType discordGameEventType = DiscordGameEventType.fromRegion(playerRegionID);
+		DiscordGameEventType discordGameEventType = DiscordGameEventType.fromRegion(playerRegionID);
```

Comment:
Add a newline after this so it doesn't look like one big block.
```suggestion
		DiscordGameEventType discordGameEventType = DiscordGameEventType.fromRegion(playerRegionID);

```